### ISSUE TITLE
foreground service automatically stops after some time

### ISSUE THREAD

**Issue by sachuss** [sachuss/Raksha-SOS-Alert-#1]

**Comment by sachuss** [comment]
https://stackoverflow.com/questions/37751823/how-to-use-firebase-eventlistener-as-a-background-service-in-android

**Comment by nikhilgeo** [comment]
> https://stackoverflow.com/questions/37751823/how-to-use-firebase-eventlistener-as-a-background-service-in-android

I don't think this fits into our usecase. Here the discussion is about starting a background service based on the change in Firebase database.

**Comment by nikhilgeo** [comment]
https://medium.com/androiddevelopers/who-lives-and-who-dies-process-priorities-on-android-cb151f39044f

**Comment by sachuss** [comment]
Using AlarmManager for checking 'if service is live or not'.
https://developer.android.com/reference/android/app/AlarmManager (Please look)
